src/rt/typeinfo/ti_real.d

```diff
@@ -66,4 +66,10 @@ class TypeInfo_e : TypeInfo
     }
 
     override @property immutable(void)* rtInfo() nothrow pure const @safe { return rtinfoNoPointers; }
+
+    static if (real.mant_dig != 64) // exclude 80-bit X87
```

Comment:
The comment doesn't seem to match the logic here.

Reply:
[80-bit extended precision floating point numbers have 64 bits of mantissa, 15 bits of exponent, and a sign bit.](https://en.wikipedia.org/wiki/Extended_precision#x86_extended_precision_format)

Reply:
Ah, derp I read `sizeof` for some reason.